bindings/java/src/main/com/apple/foundationdb/tuple/FastByteComparisons.java

```diff
@@ -205,7 +205,18 @@ public int compare(byte[] o1, byte[] o2) {
                         (PrivilegedAction<Object>) () -> {
                             try {
                                 Field f = Unsafe.class.getDeclaredField("theUnsafe");
-                                f.setAccessible(true);
+                                String javaVersion = System.getProperty("java.version");
+                                int version = Integer.parseInt(javaVersion);
+                                if(version < 13) {
```

Comment:
`canAccess` is supported since version 9, and `isAccessible` is deprecated since the same, so I think we can use that version to switch behavior.